tests/python/pytorch/sparse/test_matmul.py

```diff
@@ -98,7 +98,9 @@ def test_spspmm(create_func1, create_func2, shape_n_m, shape_k, nnz1, nnz2):
 
     torch_A1 = sparse_matrix_to_torch_sparse(A1)
     torch_A2 = sparse_matrix_to_torch_sparse(A2)
-    torch_A3 = torch.sparse.mm(torch_A1, torch_A2)
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", category=UserWarning)
+        torch_A3 = torch.sparse.mm(torch_A1, torch_A2)
```

Comment:
This is weird, torch_A1 and A2 are both sparse_coo_tensor, why the "UserWarning: Sparse **CSR tensor** support is in beta state." is triggered?

https://github.com/dmlc/dgl/blob/976970554d95d96b802faa32488d4bf836678340/tests/python/pytorch/sparse/utils.py#L153C17-L153C34

Reply:
I don't see this warning when I run this locally.

Reply:
I don't see this in my local run and also CI, are you referring to:

tests/python/pytorch/sparse/test_sparse_matrix.py::test_torch_sparse_csr_conversion[shape0-indices0-indptr0]
  /root/jenkins/workspace/dgl_PR-6147@2/tests/python/pytorch/sparse/test_sparse_matrix.py:583: UserWarning: Sparse CSR tensor support is in beta state. If you miss a functionality in the sparse tensor support, please submit a feature request to https://github.com/pytorch/pytorch/issues. (Triggered internally at ../aten/src/ATen/SparseCsrTensorImpl.cpp:54.)
    torch_sparse_csr = torch.sparse_csr_tensor(

Reply:
Actually, I see similar warnings in both places:
```
tests/python/pytorch/sparse/test_matmul.py::test_spspmm[1-1-3-shape_n_m0-rand_coo-rand_coo]
  /opt/dgl/dgl-source/tests/python/pytorch/sparse/test_matmul.py:101: UserWarning: Sparse CSR tensor support is in beta state. If you miss a functionality in the sparse tensor support, please submit a feature request to https://github.com/pytorch/pytorch/issues. (Triggered internally at /opt/pytorch/pytorch/aten/src/ATen/SparseCsrTensorImpl.cpp:53.)
    torch_A3 = torch.sparse.mm(torch_A1, torch_A2)
```
and 
```
tests/python/pytorch/sparse/test_sparse_matrix.py::test_torch_sparse_csr_conversion[shape0-indices0-indptr0]
  /opt/dgl/dgl-source/tests/python/pytorch/sparse/test_sparse_matrix.py:583: UserWarning: Sparse CSR tensor support is in beta state. If you miss a functionality in the sparse tensor support, please submit a feature request to https://github.com/pytorch/pytorch/issues. (Triggered internally at /opt/pytorch/pytorch/aten/src/ATen/SparseCsrTensorImpl.cpp:53.)
    torch_sparse_csr = torch.sparse_csr_tensor(
```
I just missed this warning `sparse/test_sparse_matrix.py` and I will fix it in a separate PR.